README.md

```diff
@@ -1,3 +1,6 @@
+# Project Overview
+This project allows a user to view and sort a number of animals (fictional and otherwise!) based on their horns.
+
```

Comment:
Hi there! 

- I think all we need to add in addition to the project overview is the collaborators section Ryan mentioned 👍